layers/+lang/haskell/funcs.el

```diff
@@ -124,7 +124,7 @@
 
 (defun haskell-intero/pop-to-repl (&optional prompt-options)
   (interactive "P")
-  (pop-to-buffer (intero-repl-buffer prompt-options)))
+  (pop-to-buffer (intero-repl-buffer prompt-options t)))
```

Comment:
Shouldn't `haskell-intero/display-repl` receive the same treatment?

Reply:
@bmag, I agree about ',ss'; as it's much easier to type, and (partially) consistent with the other language layers.

I haven't verified in-depth, but from memory, there is a bit of inconsistency of the switch to/from REPL functionality across languages.  I think it should be consistent.  A few languages layers bind ',ss' (Clojure, Racket).  Many layers bind ',si' to start the REPL.  I think in some cases ',si' this starts the REPL buffer in the background, in others it starts and switches to the REPL.

If you prefer, I can close this PR, and submit a PR that changes the keybinding for both Intero and haskell-interactive to ',ss'.

Reply:
You can just update this PR by pushing more commits (normally we prefer that you also squash them, but if that's a problem then I can do it myself).

What about `haskell-intero/display-repl`? Shouldn't its call to `intero-repl-buffer` also be modified?

Reply:
@bmag I applied the same treatment to `haskell-intero/display-repl`, and swapped ',ss' and ',sS'.